opensrp-chw/src/main/java/org/smartregister/chw/model/NavigationModel.java

```diff
@@ -21,8 +21,14 @@
     private NavigationModel() {
         navigationOptions.clear();
         NavigationOption op1 = new NavigationOption(R.mipmap.sidemenu_families, R.mipmap.sidemenu_families_active, Constants.DrawerMenu.ALL_FAMILIES, 0);
-        NavigationOption op2 = new NavigationOption(R.mipmap.sidemenu_children, R.mipmap.sidemenu_children_active, Constants.DrawerMenu.CHILD_CLIENTS, 0);
-        navigationOptions.addAll(asList(op1, op2));
+        NavigationOption op2 = new NavigationOption(R.mipmap.sidemenu_anc, R.mipmap.sidemenu_anc_active, Constants.DrawerMenu.ANC, 0);
```

Comment:
@ladleo do not overwrite the changes form the default chw app. You use conditional logic to check the version of the app and only modify for the BA/TZ build

Reply:
Updated it